src/rules/stylelint-disable-reason/README.md

```diff
@@ -0,0 +1,127 @@
+# stylelint-disable-reason
+
+Control comments should be preceded by a comment explaining why stylelint are being disabled.
+
+```css
+a {
+  /* Need for IE 6 */                   /*←*/
+  /* eslint-disable no-browser-hacks */ /*↑*/
+  /* Need for IE 6 */                   /*←*/
+  _display: block;                      /*↑*/
+  /* eslint-enable no-browser-hacks */  /*↑*/
+}                                       /*↑*/
+/**                                       ↑
+*                           The reason of disable stylelint */
+```
+
+## Options
+
+`string`: `"always-preceding"|"always-succeeding"`
```

Comment:
I'd prefer `"always-before"` and `"always-after"` because they're simpler English.